python/pyspark/ml/torch/torch_run_process_wrapper.py

```diff
@@ -0,0 +1,80 @@
+#
```

Comment:
I will need to add tests for this wrapper but I am not sure how I would unit test it... Would I put all the code inside a function called `main()` and just unit test that?

Reply:
No. You can add unit test for it. `torch_run_process_wrapper` is also a python module, in test code you can still import it.